tests/bson-corpus/binary-valid-011.phpt

```diff
@@ -1,5 +1,5 @@
 --TEST--
-Binary type: subtype 0x80
```

Comment:
Why did you used this code that isn't in the list?

Reply:
Just seen it in the tests corpus. You simply renamed some test files.

Reply:
This is due to https://github.com/mongodb/specifications/commit/db7eb39bba5eafa33f48376dbe224180c8af705e adding a test in the middle of an array. That causes [convert-bson-corpus-tests.php](https://github.com/mongodb/mongo-php-driver/blob/master/scripts/convert-bson-corpus-tests.php) to shift all of the generated `phpt` files from that point forward.